src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1020,6 +1021,33 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode, callable $e
                             ->end()
                             ->defaultValue([])
                         ->end()
+                        ->arrayNode('globals')
+                            ->info('Global parameters.')
+                            ->example(['app_version' => 3.14])
+                            ->normalizeKeys(false)
+                            ->useAttributeAsKey('name')
+                            ->arrayPrototype()
+                                ->fixXmlConfig('parameter')
+                                ->children()
+                                    ->variableNode('value')->end()
```

Comment:
This should be a scalar node, not a variable node. There is no reason to support non-scalar values as those would fail at runtime (an array cannot be casted to string to be inserted in the translation)